xcube/core/gen2/local/describer.py

```diff
@@ -71,10 +71,8 @@ def _describe_dataset(self, input_config: InputConfig) \
         try:
             descriptor = opener.describe_data(input_config.data_id,
                                               data_type=DATASET_TYPE)
-        except DataStoreError:
-            raise CubeGeneratorError(f'Data store '
-                                     f'"{input_config.store_id}" '
-                                     f'does not support datasets',
+        except DataStoreError as dse:
+            raise CubeGeneratorError(f'{dse}',
                                      status_code=400)
```

Comment:
Keep original traceback:

```suggestion
            raise CubeGeneratorError(f'{dse}',
                                     status_code=400) from dse
```